codec/marshaler_test.go

```diff
@@ -215,4 +215,20 @@ var _ = Describe("type Marshaler", func() {
 			Expect(err).Should(HaveOccurred())
 		})
 	})
+
+	Describe("func MediaTypesFor()", func() {
+		It("returns media types for the existing message type in order of codec priority", func() {
+			mt := marshaler.MediaTypesFor(reflect.TypeOf(&ProtoMessage{}))
+			Expect(mt).To(HaveLen(3))
+
+			Expect(mt[0]).To(Equal("application/vnd.google.protobuf; type=dogmatiq.marshalkit.fixtures.ProtoMessage"))
+			Expect(mt[1]).To(Equal("application/vnd.google.protobuf+json; type=dogmatiq.marshalkit.fixtures.ProtoMessage"))
+			Expect(mt[2]).To(Equal("application/json; type=ProtoMessage"))
```

Comment:
Not sure if I'm missing something, but any reason not to do this?

```suggestion
			Expect(mt).To(Equal([]string{
				"application/vnd.google.protobuf; type=dogmatiq.marshalkit.fixtures.ProtoMessage",
				"application/vnd.google.protobuf+json; type=dogmatiq.marshalkit.fixtures.ProtoMessage",
				"application/json; type=ProtoMessage",
			})
```


In the case it fails it'll produce a much nicer failure report, rather than just complaining about the length.